imports/stylesheets/event.import.styl

```diff
@@ -324,3 +324,6 @@ td.col-md-6.countCell.text-center
       li
         color #95989A
         padding 4px 0
+
+.modal-header h4.modal-title
+  line-height 2
```

Comment:
In #90 I introduced an independent file that contains general styles pertaining to modals. I think this change me be unnecessary after those changes.